raft/Raft.cfg

```diff
@@ -0,0 +1,23 @@
+\* SPECIFICATION
+\* Uncomment the previous line and provide the specification name if it's declared
+\* in the specification file. Comment INIT / NEXT parameters if you use SPECIFICATION.
+
+CONSTANTS
+    Value = {2, 3, 4}
```

Comment:
Is value restricted to numbers? I think to reduce the states, is it only necessary to distinguish it from `NoValue` and `ConfigChange`, maybe it can be a fixed value, such as `Value = "testdata"`.

Reply:
Actually the value is not used yet. Only NoValue is proposed at the moment. Using different value for testing apply correctness later.

Reply:
got it.